docs/changelog.md

```diff
@@ -26,6 +26,46 @@
 .. role:: small
 ```
 
+## 2024-08-03 {small}`lamindb 0.75.1`
+
+Features.
+
+- 🚸 In Vitessce integration, enable a Vitessce button next to a dataset, also, document valid suffixes [PR](https://github.com/laminlabs/lamindb/pull/1799) [@falexwolf](https://github.com/falexwolf)
+- ✨ Allow exclude values in Curate.validate [PR](https://github.com/laminlabs/lamindb/pull/1798) [@sunnyosun](https://github.com/sunnyosun)
+
+User experience.
+
+- 🔇 Remove warning of loading non-default source [PR](https://github.com/laminlabs/bionty/pull/32) [@sunnyosun](https://github.com/sunnyosun)
+
+Docs.
+
+- 📝 Improve curate docs [PR](https://github.com/laminlabs/lamindb/pull/1795) [@Zethson](https://github.com/Zethson)
+
+Use cases.
+
+- 📝 Mutate tiledbsoma store [PR](https://github.com/laminlabs/lamin-usecases/pull/137) [@Koncopd](https://github.com/Koncopd)
+- ♻️ Refactor and improved UX of cellxgene curate [PR](https://github.com/laminlabs/cellxgene-lamin/pull/81) [@Zethson](https://github.com/Zethson)
+- 🚸 Improved support for perturbation models | [PR](https://github.com/laminlabs/wetlab/pull/56) | [Zethson](https://github.com/Zethson)
```

Comment:
This could go under a heading `wetlab` but it'd really merit a code block that explains what it is.

Otherwise, I feel it looks like a half-cooked thing that nobody would want to use anyway.

Reply:
If this is ready for prime time for the public, the PR description should also be updated with something that's accessible to the public, shouldn't it, @Zethson?

<img width="688" alt="image" src="https://github.com/user-attachments/assets/545a8c60-f788-4349-b414-d53783abdfae">

Reply:
I wanted to write an example notebook as a use-case that we feature on the docs. This is on my TODO list. I would have announced it with that.